src/view/components/AddEntry/index.tsx

```diff
@@ -1,242 +0,0 @@
-import * as React from 'react';
```

Comment:
Why did you delete all MilkEntry pages?

Reply:
Files were untracked. They are added now